packages/babel-plugin-minify-mangle-names/README.md

```diff
@@ -71,3 +71,4 @@ require("babel-core").transform("code", {
 + `blacklist` - A plain JS Object with keys as identifier names and values indicating whether to exclude
 + `eval` - mangle identifiers in scopes accessible by eval
 + `keepFnName` - prevent mangler from altering function names. Useful for code depending on `fn.name`
++ `topLevel` - mangle topLevel Identifiers
```

Comment:
Even though most of the options currently describe the behavior not the type (i.e. "do this optimization with this stuff" vs. "this code is of such type and will be minified accordingly, whatever that implies") I think what you're suggesting makes sense.

Can we just use `sourceType` then?

| sourceType | top level mangling | unreferenced vars immutable |
|---|---|---|
| module | yes | no |
| script | no | yes |